Libraries/LibJS/Tests/Infinity-basic.js

```diff
@@ -0,0 +1,29 @@
+function assert(x) { if (!x) throw 1; }
+
+try {
+    // Parser doesn't understand "-Infinity" yet :)
```

Comment:
After https://github.com/SerenityOS/serenity/pull/1587 gets merged, I'll simplify this file:

```
NegativeInfinity -> -Infinity
(0-x) -> -x
```